app/assets/stylesheets/main.css.scss

```diff
@@ -0,0 +1,37 @@
+#container {
+  margin: 48px auto;
+  width: 960px;
+  > {
+    section {
+      clear: both;
+    }
+    header {
+      margin-bottom: 46px;
+      > h1 {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3